edb/server/protocol/binary.pyx

```diff
@@ -2230,7 +2232,10 @@ cdef class EdgeConnection:
         return out_buf
 
     def connection_made(self, transport):
-        if not self.server._accepting_connections:
+        if (
+            not self.server._accepting_connections
+            or not self.server._accept_new_tasks
```

Comment:
Same thing here. I'm not sure whether `_accepting_connections` and `_accept_new_tasks` should be exposed as a single public property/method or different ones, though.

Reply:
Looks like we don't need `_accepting_connections` any more - I've created `Server.is_accepting_connections()` based on `Server._serving`.